src/Orleans/Streams/Predicates/IStreamNamespacePredicate.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Orleans.Streams
+{
+    public interface IStreamNamespacePredicate
+    {
+        bool IsMatch(string streamNamespace);
```

Comment:
ignore this... I found out Regex is doing the same naming too. So IsMatch is good with me